Wikipedia/Code/ReadingListDetailViewController.swift

```diff
@@ -157,7 +157,7 @@ class ReadingListDetailViewController: ColumnarCollectionViewController, Editabl
     // MARK: - Empty state
     
     override func isEmptyDidChange() {
-        guard readingList.countOfEntries == 0 else {
+        guard isEmpty == (readingList.countOfEntries == 0) else {
```

Comment:
Just curious, why `==` vs `||`?

Reply:
you can try https://phabricator.wikimedia.org/T192165 with `||`, also if you swipe to delete in the detail view, then tap `Edit`, the action pane won't open

Reply:
if the collection view goes from an empty state to a nonempty state, `isEmpty || readingList.countOfEntries == 0` (`false || false`)  will be false and we'll never update `editController's` `isCollectionViewEmpty`

Reply:
Oh interesting thank you! I wonder if we should put the contents of that guard into a `let` with a name which conveys what you just explained. If you can think of something that'd be cool. No stress if not.

Reply:
Oh I noticed one more strange related edge case where the empty view doesn't appear:

![4 mov](https://user-images.githubusercontent.com/3143487/38763250-2a445d28-3f4c-11e8-99ee-c566e8f99518.gif)

Follow-on is fine if you prefer. Let me know :)

Reply:
Yeah, good point! Let me add that let and fix that bug in this PR :) thanks for checking it out!